api/Pipfile

```diff
@@ -0,0 +1,14 @@
+[[source]]
```

Comment:
Consider moving file to the "semester-project-team-2" directory since it contains the dependencies for the entire project.

Reply:
Because the back-end is the only one using the dependencies (front-end is in react so it will probably use other dependencies for whatever they need) I think it is prudent to keep the back-end and all its dependencies apart.